static/src/javascripts/projects/commercial/modules/article-aside-adverts.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import $ from 'lib/$';
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import fastdom from 'lib/fastdom-promise';
+import commercialFeatures from 'commercial/modules/commercial-features';
+
+const minArticleHeight = 1300;
+const minImmersiveArticleHeight = 600;
+
+const minContentHeight = () =>
+    config.page.isImmersive ? minImmersiveArticleHeight : minArticleHeight;
+
+const init = (start: () => void, stop: () => void) => {
+    start();
+
+    const $col = $('.js-secondary-column');
+
+    // are article aside ads disabled, or secondary column hidden?
+    if (
+        !(commercialFeatures.articleAsideAdverts &&
+            $col.length &&
+            $col.css('display') !== 'none')
+    ) {
+        stop();
+        return Promise.resolve(false);
+    }
+
+    const $mainCol = $('.js-content-main-column');
+    const $adSlot = $('.js-ad-slot', $col);
```

Comment:
These are the only real change in this file. These were defined as `let` before the `if` statement, and then defined after it.

The linter won't allow them to be `var`s that are not at the top of the block, and when moved it doesn't like them not set.
The linter won't let them be `const` because you then can't define them.
The linter won't let them be `let` because it doesn't recognise that we are actually setting them later in the code.

🤔